seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2647,6 +2647,48 @@
       }
     },
     {
+      "id": "598e8944f009e646fc236146",
+      "title": "Understanding Undefined Value returned from a Function",
+      "description": [
+        "A function can include the <code>return</code> statement but it does not have to. In case that the function does not have a <code>return</code> statement, the function will process the inner code and the returned value in that case will be <code>undefined</code>.",
+        "<strong>Example</strong>",
+        "<blockquote>var sum = 0; function addSum(num) {<br>  sum = sum + num;<br>}<br>var returnedValue = addSum(3); // sum will be modified but returned value is undefined</blockquote>",
+        "<code>addSum</code> is a function without a <code>return</code> statement. The function will change the global <code>sum</code> variable but the returned value of the function is <code>undefined</code>",
+        "<hr>",
+        "Create a function <code>addFive</code> without any argument. This function add 5 to the <code>sum</code> variable, but its returned value is <code>undefined</code>."
```

Comment:
Minor tweak: the second sentence should say "This function adds 5..." (vs. "add 5")